### ISSUE TITLE
Upgrade pa11y to v7

### ISSUE THREAD

**Issue by rcantin-w** [wellcomecollection/wellcomecollection.org#10491]
We want to test against the latest WCAG so we should upgrade. The move to 6 proved difficult because of old dependencies within it, which 7 aimed to fix.

See PRs #10468 #10483 and #10484  for what was tried. 

[It should be released any day](https://github.com/pa11y/pa11y/pull/681) (they had aimed for 5/12)

**Comment by rcantin-w** [comment]
in prod
